src/AutoRest.CSharp/MgmtTest/Generation/ResourceCollectionTestWriter.cs

```diff
@@ -72,6 +72,8 @@ private void WriteUsings(CodeWriter writer)
             writer.UseNamespace("System");
             writer.UseNamespace("System.Threading.Tasks");
             writer.UseNamespace("System.Net");
+            writer.UseNamespace("Azure");
+            writer.UseNamespace("Azure.Core");
```

Comment:
This should be redundant if we are writing the type using interpolation like `$"{typeof(WaitUntil)}"`.
```suggestion
```